circle.yml

```diff
@@ -0,0 +1,22 @@
+machine:
+  java:
+    version: oraclejdk7
+  environment:
+    TERM: dumb
+
+general:
+  artifacts:
+    - error-handling/build/libs
+    - http-clients/build/libs
+    - http-servers/build/libs
+    - ssl-config/build/libs
+
+test:
+  override:
+    - ./gradlew build
+
+deployment:
+  release:
```

Comment:
Other projects have a "develop" block that runs a "publishToArtifactory" Gradle task, but this project doesn't have that task